CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 0.2.0
+
+##### Breaking
+
+* Jazzy now only supports projects using Swift 1.2.  
+  [JP Simard](https://github.com/jpsim)
+  [#170](https://github.com/realm/jazzy/issues/170)
+* Increase default minimum ACL to public.  
+  [JP Simard](https://github.com/jpsim)
+  [#186](https://github.com/realm/jazzy/issues/186)
+
+##### Enhancements
+
+* Use key.accessibility to determine ACL.  
```

Comment:
either surround `key.accessibility` in backticks, or describe the _what_ of the change rather than the _how_